docs/ReferenceOneField.md

```diff
@@ -18,7 +18,7 @@ This field fetches a one-to-one relationship, e.g. the details of a book, when u
                        └──────────────┘
 ```
 
-`<ReferenceOneField>` behaves like `<ReferenceManyField>`: it uses the current `record` (a book in this example) to build a filter for the book details with the foreign key (`book_id`). Then, it uses `dataProvider.getManyReference('book_details', { target: 'book_id', id: book.id })` to fetch the related details, and takes the first one.
+`<ReferenceOneField>` behaves like `<ReferenceManyField>`: it uses the current `record` (a book in this example) to build a filter for the book details with the foreign key (`book_id`) and optional additional filters. Then, it uses `dataProvider.getManyReference('book_details', { target: 'book_id', id: book.id })` to fetch the related details, and takes the first one.
```

Comment:
Same, let's not mention the 'optional additional filters' so early, and keep them for later in the doc.